data/interfaces/default/info.html

```diff
@@ -361,11 +374,122 @@ <h4 class="modal-title" id="myModalLabel">Confirm Delete</h4>
+                        <div class="modal-dialog">
+                            <div class="modal-content">
+                                <div class="modal-header">
+                                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true"><i class="fa fa-remove"></i></button>
+                                    <h4 class="modal-title" id="myModalLabel">Confirm Update</h4>
+                                </div>
+                                <div class="modal-body" style="text-align: center;">
+                                    <p>Are you REALLY sure you want to replace
+                                    <p><strong>
+                                        % if query['media_type'] == 'movie':
+                                        ${query['title']}<br />${query['year']}
+                                        % elif query['media_type'] == 'show':
+                                        ${query['grandparent_title']}
+                                        % elif query['media_type'] == 'season':
+                                        ${query['grandparent_title']}<br />S${query['parent_media_index']}
+                                        % elif query['media_type'] == 'episode':
+                                        ${query['grandparent_title']}<br />${query['title']}<br />S${query['parent_media_index']} &middot; E${query['media_index']}
+                                        % elif query['media_type'] == 'artist':
+                                        ${query['grandparent_title']}
+                                        % elif query['media_type'] == 'album':
+                                        ${query['grandparent_title']}<br />${query['parent_title']}
+                                        % elif query['media_type'] == 'track':
+                                        ${query['grandparent_title']}<br />${query['title']}<br />${query['parent_title']}
+                                        % endif
+                                        </strong></p>
+                                    <p> with </p>
+                                    <p><span id="new_title"></span></p>
+                                    % if query['media_type'] != 'movie':
+                                    <p>All items for <strong>${query['grandparent_title']}</strong> will also be updated.</p>
+                                    % endif
+                                    <p>This is permanent and cannot be undone!</p>
+                                </div>
+                                <div class="modal-footer">
+                                    <button type="button" class="btn btn-dark" data-dismiss="modal">Cancel</button>
+                                    <button type="button" class="btn btn-danger btn-ok" data-dismiss="modal" id="confirm-update">Update</button>
```

Comment:
Can we get this button to disable once clicked - not sure of what unexpected results may happen if you issue this request multiple times.